tailwind.config.js

```diff
@@ -7,8 +7,39 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
-  },
+    colors: {
+      'black': '#1c1c1c',
+      'borders': "#DDDDDD",
+      'bg':'#F3F5F7',
+      'danger-aler-bg': '#FC555619',
+      'unAssigned-bg': "#F7C2D2",
+      'Assigned-bg': "#99BFFB",
+      'InProgress-bg': "#FFE28C",
+      'Completed-bg': "#A1EDB1",
+      'placeholder':'#4D4D4D',
+      'white': '#ffffff',
+      'border-color': '#464646',
```

Comment:
this is not being used; Even if you are using it, give a meaningful color name;
Not abstract names like border-color.
What if you are using it in places other than border ?